conveyor/src/main/java/com/enfint/conveyor/Main.java

```diff
@@ -0,0 +1,23 @@
+package com.enfint.conveyor;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.client.RestTemplate;
+
+
+
+@SpringBootApplication
+public class Main {
+
+    @Bean
+    public RestTemplate restTemplate(){
+        return new RestTemplate();
+
+    }
```

Comment:
it would be better if this bean-defenition will be located in separate class, marked @Configuration